meilisearch-release-changelog/v0.30.0.md

```diff
@@ -0,0 +1,229 @@
+# v0.30.0 release changelogs
+
+<!-- The following line should NOT be put in the official release changelogs -->
+⚠️ Since this is a release candidate (RC), we do NOT recommend using it in a production environment. Something not working as expected? We welcome [bug reports](https://github.com/meilisearch/meilisearch/issues/new/choose) and [feedback about new features](https://github.com/meilisearch/product/discussions).
+
+This release removes index deployment downtime and adds new task management features. It also introduces new search parameters that facilitate the creation of numbered pagination interfaces.
+
+Thanks to Hacktoberfest, tons of updates and bug fixes have been done by the fantastic Meilisearch community. You were more present than ever and words cannot make justice tojust  how grateful we are for your support. A big thank you from the whole Meilisearch team! ❤️
+
+<!-- The following line should NOT be put in the official release changelogs -->
+📖 Meilisearch's official documentation will be ready for the official v0.30.0 release. Meanwhile, work-in-progress documentation is available [here]().
+
+# New features and improvements 🔥
+
+## Ease search results navigation with pagination
```

Comment:
Need input of @gmourier here

Reply:
IMO, pagination implies a number; I find it a bit clumsy to say "Numbered pagination". The infinite scroll or load more methods are ways to navigate the search results, just like pagination, which is a different method.

Reply:
I agree w/ Gui, "ease search result navigation" is not very clear.

Gui's suggestion is good, and matches the "verb first" formula followed by other features in the changelog ("delete old processed tasks", "remove downtime").

I disagree that "numbered pagination" is redundant, because unnumbered pagination also exists ("next page / previous page").

Perhaps something like this would work as a compromise?

`Improve search result navigation: create numbered pagination interfaces`

Reply:
Thanks for the clarification, @guimachiavelli and @dichotommy. It makes sense!

I like reading `search result navigation` in the title because I know it relates to navigating search results at a glance. It's clearer to me 🎯

Reply:
ok, that sounds good. I'll make the suggestion based on our discussion, then.

```suggestion
## Improve search result navigation: create numbered pagination interfaces
```